frontend/awx/views/schedules/wizard/ScheduleEditWizard.tsx

```diff
@@ -49,7 +49,12 @@ export function ScheduleEditWizard() {
     await Promise.resolve();
   };
 
-  const onCancel = () => navigate(location.pathname.replace('edit', 'details'));
+  const onCancel = () =>
+    navigate(
+      params?.id
+        ? location.pathname.replace('edit', 'details')
+        : `/templates/job_template/${schedule?.unified_job_template}/schedules/${schedule?.id}/details`
```

Comment:
Do we need to cover workflow_job_template as well?

Reply:
Good catch! I'll update it.
Is job template and workflow job template the only two resource types when need to accommodate for the cancel button?

Reply:
@AlexSCorey Do you know this one?